ace.d.ts

```diff
@@ -953,6 +953,8 @@ export namespace Ace {
     splitLine(): void;
     setGhostText(text: string, position: Point): void;
     removeGhostText(): void;
+    setWidgetManager(): void;
```

Comment:
Do we need to expose this function? consumers will want to access the widget manager (which they can do through `getWidgetManager`) and when it's not defined yet that will be handled by this getter. Do you see an use-case where someone wants to attach a widget manager but not use it afterwards?